resolvers/webpack/index.js

```diff
@@ -44,13 +44,15 @@ exports.resolve = function (source, file, settings) {
 
   var webpackConfig
 
-  var configPath = get(settings, 'config')
+  var _configPath = get(settings, 'config')
     , configIndex = get(settings, 'config-index')
     , env = get(settings, 'env')
     , argv = get(settings, 'argv', {})
     , packageDir
 
-  log('Config path from settings:', configPath)
+  var configPath = typeof _configPath === 'string'
+     ? _configPath.replace(/^\.(?=[/\\])/, path.resolve(path.join(__dirname, '..', '..')))
```

Comment:
I mean, node's path helpers should "just work" with `path.relative(process.cwd(), path)` - if `path` is absolute, it will lead with a `/`, otherwise it will be relative (the common case). Why is a placeholder needed?

Reply:
for a `.eslintrc` in json5, instead of a `.eslintrc.js`

Reply:
no, what i mean is, the static path in the config would either start with a dot or a slash, and the rule code would use `path.relative`.

Reply:
I'm not sure I understood, but I used path.resolve, and it looks much better